core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -17,15 +17,57 @@ final case class NonEmptyList[+A](head: A, tail: List[A]) {
 
   /**
    * Return the head and tail into a single list
+   * {{{
+   * scala> import cats.data.NonEmptyList
+   * scala> val nel = NonEmptyList.of(1, 2, 3, 4, 5)
+   * scala> nel.toList
+   * res0: scala.collection.immutable.List[Int] = List(1, 2, 3, 4, 5)
+   * }}}
    */
   def toList: List[A] = head :: tail
 
+  /**
+   * Selects the last element
+   * {{{
+   * scala> import cats.data.NonEmptyList
+   * scala> val nel = NonEmptyList.of(1, 2, 3, 4, 5)
+   * scala> nel.last
+   * res0: Int = 5
+   * }}}
+   */
   def last: A = tail.lastOption match {
     case None    => head
     case Some(a) => a
```

Comment:
@johnynek yeah, I forgot that getOrElse is by-name, and corrected myself in the other comment https://github.com/typelevel/cats/pull/1628#discussion_r113015121